### ISSUE TITLE
prettify app hang

### ISSUE THREAD

**Issue by vaclav18** [pmahend1/PrettyXML#88]
Steps to reproduce:
- install Pretty XML (https://code.visualstudio.com/docs/editor/extension-marketplace)
- switch to a tab containing a minified XML
- try to preffify it
Result:
No any prettify, a progress bar (possibly an infinite loop)
Plattform: Linux, Latest VS Code

**Comment by pmahend1** [comment]
Thanks for reporting. I think you are missing C# extension and/or dotnet in path.
Could you please follow these steps and let me know if that solves for you?

https://github.com/pmahend1/PrettyXML/issues/64#issuecomment-959951682

**Comment by vaclav18** [comment]
> Thanks for reporting. I think you are missing C# extension and/or dotnet in path. Could you please follow these steps and let me know if that solves for you?
> 
> [#64 (comment)](https://github.com/pmahend1/PrettyXML/issues/64#issuecomment-959951682)

Hi, Thank you for the explanation.
With full respect: an additional piece of code required (like a dependency) should have been evaluated and reported to the user.
I'm switching to a different extension without this dependecy (It'd more easy to boot Windows and use Visual Studio instead).
As your users probably prefer a Windows environment I suppose you simply cannot invest too much time into this one.

Bottom line: Thank you for you time and please don't take this one any wrong way.
Best
Vaclav

**Comment by pmahend1** [comment]
The infinite formatting progress has been fixed now.
This extension has hard dependency on .Net SDK unfortunately. It was mentioned in ReadMe, and I have [updated it to be clearer](https://github.com/pmahend1/PrettyXML#requirements).  .Net can be installed on Mac and [Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux) , so this is not targeted to just Windows.

Appreciate your feedback since I added Linux support few months back.
